doc/usage_rsmxval.rst

```diff
@@ -0,0 +1,77 @@
+.. _usage_rsmxval:
+
+``rsmxval`` - Run cross-validation experiments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+RSMTool provides the ``rsmxval`` command-line utility to run cross-validation experiments with scoring models. Why would one want to use cross-validation rather than just using the simple train-and-evaluate loop provided by the ``rsmtool`` utility. Using cross-validation can provide more accurate estimates of scoring model performance since those estimates are averaged over *multiple* train-test splits that are randomly induced from the data. Using a single train-test split may lead to biased estimates of performance since those estimates will depend on the specific characteristics of that split. Using cross-validation is more likely to provide estimates of how well the scoring model will generalize to unseen test data, and more easily flag problems with overfitting and selection bias, if any. 
+
+Cross-validation experiments in RSMTool consist of the following steps:
+
+1. The given training data file is first shuffled randomly (with a fixed seed for reproducibility) and then split into the requested number of folds.  It is also possible for the user to provide a CSV file containing a pre-determined set of folds, e.g., from another part of the data pipeline.
+
+2. For each fold (or train-test split), ``rsmtool`` is run to train a scoring model on the training split and evaluated on the test split. All of the outputs for each of the ``rsmtool`` runs is saved on disk and represents the per-fold performance. 
+
+3. The predictions generated by ``rsmtool`` for each of the folds are all combined into a single file which is then used as input for ``rsmeval``. The output of this evaluation run is saved to disk and provides a more accurate estimate of the predictive performance of a scoring model trained on the given data. 
+
+4. A summary report comparing all of the folds is generated by running ``rsmsummarize`` on all of the fold directories created in the Step 1 and its output is also saved to disk. This summary output can be useful to see if the performance for any of the folds stands out for any reason which could point to a potential problem. 
+
+5. Finally, a scoring model is trained on the complete training data file using ``rsmtool`` which also generates report that contains only the feature and model descriptives. The model is what will most likely be deployed for inference assuming the analyses produced in this step and Steps 1--4 meet the stakeholders' requirements.
+
+.. include:: tutorial_rsmxval.rst
+
+Input
+"""""
+
+```rsmxval`` requires a single argument to run an experiment: the path to :ref:`a configuration file <config_file_rsmxval>`. It can also take an output directory as an optional second argument. If the latter is not specified, ``rsmeval`` will use the current directory as the output directory. 
```

Comment:
```suggestion
```rsmxval`` requires a single argument to run an experiment: the path to :ref:`a configuration file <config_file_rsmxval>`. It can also take an output directory as an optional second argument. If the latter is not specified, ``rsmxval`` will use the current directory as the output directory. 
```